apps/studio/src/features/editing-experience/components/LayoutSelection.tsx

```diff
@@ -0,0 +1,281 @@
+                            }}
+                            _groupHover={{
+                              opacity: 0.6,
+                            }}
+                          />
+                          <VStack
+                            sx={{
+                              transition: 'opacity 300ms ease-out',
+                              opacity: 0,
+                            }}
+                            className="hover-text"
+                            w="100%"
+                            h="100%"
+                            position="absolute"
+                            top="0"
+                            left="0"
+                            justify="center"
+                            gap="0.25rem"
+                            opacity="0"
+                            color="base.content.brand"
+                            _groupHover={{
+                              opacity: 1,
+                            }}
+                          >
+                            <Icon as={BiShow} />
+                            <Text textStyle="caption-1">Click to preview</Text>
+                          </VStack>
+                        </>
+                      )}
+                    </Box>
+                    <HStack w="100%" justifyContent="space-between">
+                      <Text
+                        className="layout-title"
+                        textAlign="left"
+                        textStyle="h6"
+                        textColor={
+                          selectedLayout.layoutTypename ===
+                          layoutEntry.layoutTypename
+                            ? 'base.content.brand'
+                            : 'base.content.strong'
+                        }
+                        mb="0.5rem"
+                        sx={{ transition: 'color 300ms ease-out' }}
+                        _groupHover={{ textColor: 'base.content.brand' }}
+                      >
+                        {layoutEntry.layoutDisplayName} layout
+                      </Text>
+                    </HStack>
+                    <Text
+                      w="100%"
+                      textAlign="left"
+                      textStyle="body-1"
+                      textColor="base.content.default"
+                    >
+                      {layoutEntry.layoutDescription}
+                    </Text>
+                  </VStack>
+                )
+              })}
+            </VStack>
+          </GridItem>
+          <GridItem
+            w="100%"
+            colSpan={3}
+            bg="grey.100"
+            p="2rem 2rem 0 2rem"
+            overflow="auto"
+          >
+            <Box
+              borderRadius="8px 8px 0px 0px"
+              w="100%"
+              minH="100%"
+              boxShadow="md"
+              position="relative"
+            >
+              <HStack
+                borderRadius="8px 8px 0px 0px"
+                w="100%"
+                bgColor="slate.200"
```

Comment:
comment: where are these colors coming from? I do not see any theme generated here. Using default chakra colors? Highly suggest using design system tokens or it'll be a PITA in the future to update colors.

Reply:
for context - we aim to refer by tokens because it provides a centralised place for our theme. the benefit is that when we change 1 colour in our theme, our whole site will be updated accordingly. 

in here, we've used an **inbuilt palette** (noticed how it's suffixed with `.200`). this isn't meant to be used to styled components and when we change our own theme (things like: `bg.focused.strong`), we'll miss out on this `HStack`!